core/app/modules/kit-library/assets/js/components/sort-select.js

```diff
@@ -35,6 +46,10 @@ export default function SortSelect( props ) {
 						icon={ 'asc' === props.value.direction ? 'eicon-arrow-up' : 'eicon-arrow-down' }
 						className="eps-sort-select__button"
 						onClick={ () => {
+							const direction = 'asc' === props.value.direction ? 'desc' : 'asc';
```

Comment:
Will not work.
If the direction is `asc` it will be `desc`.

Also, do we need this? Can't we just use the prop directly?